prow/config.yaml

```diff
@@ -436,6 +436,36 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
   GoogleCloudPlatform/k8s-multicluster-ingress:
+  - name: ci-kubemci-image-push
```

Comment:
you probably don't want it to be a presubmit job..

Reply:
Right!! Thanks for pointing it out!
Moved to postsubmit.